hr_timesheet_auto_creation/tests/test_timesheet_auto_creation.py

```diff
@@ -0,0 +1,47 @@
+# © 2016-2017 Elico Corp (https://www.elico-corp.com)
+# Copyright 2019 Coop IT Easy SC
+#   - Vincent Van Rossem <vincent@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from datetime import date, timedelta
+
+from odoo.tests import common
+
+
+class TestHrTimesheetSheet(common.TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.env["hr.employee"].search([]).write({"user_id": False})
```

Comment:
Because of https://github.com/odoo/odoo/issues/129620 , this causes a singleton ValueError.

In order to circumvent the error, do the following:

```suggestion
        # The loop is a workaround for https://github.com/odoo/odoo/issues/129620
        # Remove loop when bug is fixed upstream.
        for record in self.env["hr.employee"].search([]):
            record.write({"user_id": False})
```